src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -1894,12 +1894,12 @@ A more realistic example might have several tasks on each branch. The
 ``bar``, but configured differently to avoid the failure.
 
 
-Message Trigger Example
-^^^^^^^^^^^^^^^^^^^^^^^
+Custom Outputs
+^^^^^^^^^^^^^^
 
-Branching is particularly powerful when using :ref:`MessageTriggers` (i.e.
-:term:`custom outputs <custom output>`) to define multiple parallel paths in
-the graph.
+Branching is particularly powerful when using
+:term:`custom outputs <custom output>` to define multiple parallel paths in the
```

Comment:
```suggestion
:term:`custom outputs <custom output>` to define alternate parallel paths in the
```

(not sure, but we probably want to distinguish this from "normal" multiple parallel paths in the graph?)